### ISSUE TITLE
Pterodactyl slskd Egg

### ISSUE THREAD

**Issue by fucksophie** [slskd/slskd#750]
Recently, I finished writing my slskd egg for Pterodactyl!

It'd be cool if it would be included in pterodactyl or at least noted down somewhere in the docs..

(it's a json file, i had to change it to .txt so it would upload to Github)

[egg-slskd.txt](https://github.com/slskd/slskd/files/9995180/egg-slskd.txt)

**Comment by jpdillingham** [comment]
Thanks for doing this!  I can add it to the docs, but I'd like to confirm a few things so I can include context.  Is [this](https://pterodactyl.io/) the Pterodactyl you're talking about?

I see the config uses a generic base image and then installs slskd with a script.  Is there a reason you couldn't use the official slskd container?

**Comment by yourfriendoss** [comment]
Yes, that is the pterodactyl I'm talking about. Also, generically default docker images don't work with pterodactyl, since pterodactyl is kind of a pain to run without custom images and fixes related to pterodactyl. Also, just the fact that you can edit the install scripts and quickly iterate is quite a big plus too.
